### PR TITLE
AO3-5043 Remove admin notifications feature

### PR DESCRIPTION
**Pull request by sarken**
## Issue

https://otwarchive.atlassian.net/browse/AO3-5043

## Purpose

This removes the admin notifications feature and the corresponding "turn off admin emails" preference. It also removes the alphabet helper, which was used by the admin notifications feature and  views/people/show.html.erb. views/people/show.html.erb is being removed here because it's not in use and was otherwise preventing the removal of the alphabet helper.

## Testing

Refer to JIRA.

### CODE REVIEW

File: app/views/people/show.html.erb [hunk 1]
```diff
@@ -1,23 +0,0 @@
-<!--SEARCHBROWSE Descriptive page name, messages and instructions-->
```
Comment (zz9pzza) [inline app/views/people/show.html.erb]: Should this file be removed ( it doesn't look relevant to the pull ? )

Comment (sarken) [inline app/views/people/show.html.erb]: From the "purpose" bit:
> It also removes the alphabet helper, which was used by the admin notifications feature and views/people/show.html.erb. views/people/show.html.erb is being removed here because it's not in use and was otherwise preventing the removal of the alphabet helper.

Comment (zz9pzza) [inline app/views/people/show.html.erb]: Sorry I was thinking about index rather than show.

Comment (sarken) [inline app/views/people/show.html.erb]: No worries!
